packages/dds/tree/src/shared-tree-core/README.md

```diff
@@ -16,4 +16,4 @@ it just provides an index contract / interface which can be used for that purpos
 
 # Status
 
-Implementation is really just some notes/ideas, and nothing close to usable.
+Implementation is working, but how indexes relate to branches needs more thought. More detail on this in the [package readme](../../README.md).
```

Comment:
```suggestion
Implementation is working, but how indexes relate to branches needs more thought out. More detail on this in the [package readme](../../README.md).
```

Reply:
I think this is correct how it was. If there was a "to be" in there your change would work.

Reply:
Oh yes, I read this wrong.